packages/element-app/src/components/Snackbar/index.js

```diff
@@ -1,5 +0,0 @@
-import Snackbar from './Snackbar';
```

Comment:
Not sure what the best place to put this is, but the new snackbar logic causes some prop validation to fail. For example in FullNodeExplorerTransactionPage:
<img width="1440" alt="Screenshot 2020-01-24 at 14 08 08" src="https://user-images.githubusercontent.com/7913565/73050667-2b9d0400-3eb3-11ea-818c-8a25f285c54a.png">

Reply:
great catch.

Reply:
Solved by making sure not to set a prop to undefined...

```
  componentDidMount() {
    this.setState({
      expanded: this.props.expanded || false,
    });
  }
```